doc/guides/writing-tests.md

```diff
@@ -232,8 +232,8 @@ For performance considerations, we only use a selected subset of ES.Next
 features in JavaScript code in the `lib` directory. However, when writing
 tests, for the ease of backporting, it is encouraged to use those ES.Next
 features that can be used directly without a flag in [all maintained branches]
-(https://github.com/nodejs/lts), you can check [node.green](http://node.green)
-for all available features in each release.
+(https://github.com/nodejs/lts), [node.green](http://node.green) can be used
```

Comment:
While we're in here, fix the comma splice? (The comma here should be a period and `node.green` is the start of a new sentence.)

For that matter, maybe eliminate the passive voice in that new sentence and add a normative trailing `/` to the URL?

> [node.green](http://node.green/) lists available features in each release.